pootle/core/search/backends/__init__.py

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+from .elastic_search import *
```

Comment:
There where when I had them at the same level, haven't checked again.  The Elasticsearch module provides and Elasticsearch() class which was causing problems.